src/applications/vaos/reducers/newAppointment.js

```diff
@@ -94,12 +98,10 @@ function getFacilities(state, typeOfCareId, vaParent) {
   return state.facilities[`${typeOfCareId}_${vaParent}`] || [];
 }
 
-function getSystemFromFacility(facilities, facilityId) {
-  const facilityInfo = facilities.find(
-    facility => facility.institutionCode === facilityId,
-  );
-
-  return facilityInfo?.rootStationCode;
+// Only use this when we need to pass data that comes back from one of our
+// services files to one of the older api functions
+function parseFakeFHIRId(id) {
```

Comment:
Should we put this in a service or some other utils file since it seems like we might be using it frequently?

Reply:
It should go away once we're switched over to using the mapped FHIR data, so I think I'd rather leave it here where we know something like this wouldn't belong long term.